metadata.yaml

```diff
@@ -26,3 +26,12 @@ requires:
 provides:
   kafka-client:
     interface: kafka_client
+
+storage:
+  log-data:
+    type: filesystem
+    description: Directories where the log data is stored
+    minimum-size: 50M
+    location: /var/snap/kafka/common
+    multiple:
+      range: "12"
```

Comment:
Why is this needed?

Reply:
The multiple?

Requirement from product. Idea is that it helps parallelize writes. Some recommendations online for a production system suggest having 12 dirs as a good baseline.

Reply:
Thanks for explaining!

Reply:
What if we put 1-12? I suppose this would allow the charm to work in the "unsupported" case of having 1 storage, and then add storage up to 12 maximum slots with "add-storage"

Reply:
We can address that in later PRs once we have a clearer idea of how the workload should handle such events.